exercises/palindrome-products/example.rs

```diff
@@ -0,0 +1,33 @@
+pub type Palindrome = u64;
+pub fn get_palindrome_products(min: u64, max: u64) -> Vec<Palindrome>{
+    let mut palindromes:Vec<u64> = Vec::new();
+    for i in min..max+1{
+        for j in min..max+1{
+            if is_palindrome(i*j){
+                palindromes.push(i*j);
+            }
+        }
+    }
+    palindromes
+}
+
+pub fn min(palindromes: Vec<Palindrome>)->Option<Palindrome>{
+    if palindromes.is_empty(){
+        return None;
+    }
+    palindromes.iter().cloned().min()
+}
+
+pub fn max(palindromes: Vec<Palindrome>)->Option<Palindrome>{
+    if palindromes.is_empty(){
+        return None;
```

Comment:
is it necessary? https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.max already returns `None` if empty.